buildmodel/commands.go

```diff
@@ -397,6 +417,14 @@ type UpdateFlags struct {
 	skipSubmoduleUpdate *bool
 }
 
+type GHAuthFlags struct {
+	githubPAT             *string
+	githubPATReviewer     *string
+	gitHubAppClientID     *string
```

Comment:
Consistency?

```suggestion
type GitHubAuthFlags struct {
	gitHubPAT             *string
	gitHubPATReviewer     *string
	gitHubAppClientID     *string
```

Reply:
Hmm, but I didn't expect the PAT and PATReviewer to be in one struct. I expected one struct to be used per user/app.